src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2120,7 +2119,8 @@ static bool ride_get_place_position_from_screen_position(int32_t screenX, int32_
         if (gInputPlaceObjectModifier & PLACE_OBJECT_MODIFIER_COPY_Z)
         {
             TileElement* tileElement;
-            get_map_coordinates_from_pos(screenX, screenY, 0xFCCA, &mapX, &mapY, &interactionType, &tileElement, &viewport);
+            get_map_coordinates_from_pos(
+                screenCoords.x, screenCoords.y, 0xFCCA, &mapX, &mapY, &interactionType, &tileElement, &viewport);
```

Comment:
You can start using the new ScreenCoords and CoordsXY version of `get_map_coordinates_from_pos`

Reply:
I will, thought it hadn't been merged yet.